.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -44,6 +44,6 @@ Search open/closed issues before submitting since someone might have asked the s
 | Software          | Version(s) |
 | ----------------- | ---------- |
 | tamr-unify-client |
```

Comment:
Assume this could not be changed to remove "unify", correct?

Reply:
That's right, I'm changing all the prose to remove `Unify`, but anything that references the actual Python package on PyPI (like this) still refers to it via the actual package name `tamr-unify-client`.

As part of a 1.0 release, we will change the package name to `tamr-client`. See #74 for more about docs changes that will need to happen for a 1.0 release.

Reply:
@JuliaMalkin also note this PR duplicates commits from #297 , so you can just look at the last 3 commits for this PR. You can use this link to filter down to just the 3 relevant commits for this PR: https://github.com/Datatamer/tamr-client/pull/298/files/f8618e522a805530a3c3b2b2c3b5030d524f86c9..0f9de260deafc63a967ff7a25960f5d99b968306